.gitignore

```diff
@@ -1,2 +1,5 @@
 .*.sw?
 /node_modules/
+/src/server/webpack-stats.json
+/public/assets/*
+npm-debug.log
```

Comment:
I think we're safe to ignore `npm-debug.log` anywhere it occurs ( `*.log` might even be safe if you felt zealous).